src/main/java/step2/view/LottoOutput.java

```diff
@@ -0,0 +1,46 @@
+package step2.view;
+
+import step2.model.Lotto;
+import step2.model.LottoGame;
+import step2.model.WinnerTier;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class LottoOutput {
+
+    private LottoOutput() {
+    }
+
+    ;
+
+    private static final String LOTTO_NUMBER_DELEVETER = ",";
+
+    public static void printLottoNumbers(LottoGame lottoGame) {
+
+        List<Lotto> lottos = lottoGame.getLottos();
+
+        for (Lotto lotto : lottos) {
+
+            String lottoNumbers = lotto.getLottoNumbers().stream()
+                    .map(i -> i.toString())
+                    .collect(Collectors.joining(LOTTO_NUMBER_DELEVETER));
+
+            System.out.printf("[%s]\n", lottoNumbers);
+        }
```

Comment:
메소드 시작에 공백라인을 두지 않는 것이 일반적입니다. 공백 라인을 넣는다면 의미를 가지고 추가하면 좋을 것 같아요 :)
```suggestion
        List<Lotto> lottos = lottoGame.getLottos();
        for (Lotto lotto : lottos) {
            String lottoNumbers = lotto.getLottoNumbers().stream()
                    .map(i -> i.toString())
                    .collect(Collectors.joining(LOTTO_NUMBER_DELEVETER));
            System.out.printf("[%s]\n", lottoNumbers);
        }
```